### ISSUE TITLE
delete of used volume returns ok, but volume is not removed

### ISSUE THREAD

**Issue by lumir-mrkva** [swarmpit/swarmpit#140]
`docker volume rm` returns 

```
Error response from daemon: unable to remove volume: remove monitoring-grafana: volume is in use - [fe01b7c89369714f95b3de1ecac3772164e87ccccca5b96191b869a7be80574d, 758c02f61f40845f17bb3c5ad9caca52eab740593be3d160e4dda1a526a87269]
```

probably bug in error handling

**Comment by lumir-mrkva** [comment]
same behaviour is on other delete actions

**Comment by nohaapav** [comment]
it happens just for used resources

**Comment by nohaapav** [comment]
@lumir-mrkva Problem was introduced with new docker client:

`(throw (let [data (some-> exception (ex-data))
                   error (if (= "application/json" ((:headers data) "Content-Type"))
                           (parse-string (:body data) true)
                           {:body (:reason-phrase data)})]`

The response must be in format 
`{:status http_code
 :body   response_body (in case of error error should be in error: attribute)}`

**Comment by lumir-mrkva** [comment]
fixed by #144
